R/loop_tasks.R

```diff
@@ -31,17 +31,22 @@
 #'   failed task. Especially useful if the error was likely to be inconsistent
 #'   (e.g., a temporary network issue) and might not occur again if we wait a
 #'   while
-#' @param vebose define the format of task messages. Use TRUE for progress bar 
+#' @param verbose define the format of task messages. Use TRUE for progress bar 
 #'   for the status of each task, and FALSE for no output
+#' @param n_cores integer How many cores should be utilized when executing the task plan? 
+#' Defaults to one (no parallelization).
 #' @export
 #' @import progress
+#' @import foreach
```

Comment:
I think we either want `Imports: foreach`  and `@import foreach` or `Suggests: foreach` and `requireNamespace('foreach', quietly=TRUE)`